test/powershell/Modules/Microsoft.PowerShell.Utility/JsonObject.Tests.ps1

```diff
@@ -0,0 +1,53 @@
+﻿# Unit tests for JsonObject
+Describe 'JsonObject' -tags "CI" {
+
+    function ShouldThrow
+    {
+        param (
+            [Parameter(ValueFromPipeline = $true)]
+            $InputObject,
+            [Parameter(Position = 0)]
+            $expectedException
+        )
+
+        try
+        {
+            & $InputObject
+        }
+        catch
+        {
+            $ex = $_.Exception.InnerException
+        }
+
+        $ex | Should Not BeNullOrEmpty
+        $ex.GetType() | Should Be $expectedException
+    }
+
+    It 'empty string' {
+        $errRecord = $null
+        [Microsoft.PowerShell.Commands.JsonObject]::ConvertFromJson("", [ref]$errRecord)
```

Comment:
An example of `-testcases` can be found [here](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#parameter-generation). It's helpful to avoid duplication when there are a lot `It` blocks with same logic but different test data.

Since the number of test cases with the same logic is not many (3 for `$errRecord | Should BeNullOrEmpty` and 2 for `ShouldThrow`), I don't insist on changing the existing test cases.